roles/otlc_ocp4_installation_install/defaults/main.yml

```diff
@@ -7,5 +7,7 @@ epel_repo_gpg_key_url: '/etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-{{ ansible_distributio
 epel_repofile_path: '/etc/yum.repos.d/epel.repo'
 
 ocp_availability_zones: [ a, b, c ]
-ocp_nodes_per_zone: 2
```

Comment:
I think the ocp_nodes_per_zone and ocp_nodes_total are not used anymore and can be cleaned up?